_security-analytics/threat-intelligence/getting-started.md

```diff
@@ -0,0 +1,85 @@
+title: Getting started
+parent: Threat intelligence
+nav_order: 41
+---
+
+# Getting started
+
+To get started with threat intelligence, you'll need to set up your threat intelligent sources and set up monitors to scan your log sources. The following tutorial describes how you can get started using OpenSearch Dashboards. Alternatively, you can use the [API](({{site.url}}{{site.baseurl}}/security-analytics/threat-intelligence/api/threat-intel-api/). 
+
+## Threat intelligence view
+
+To access threat intelligence, log into OpenSearch Dashboards, and select **Security Analytics** > **Threat Intelligence**.
+
+In the threat intelligence view, you can access the following tabs:
+
+- **Threat intel sources**: Shows a list of all active and inactive threat intelligence sources, including the default IP reputation feed [AlienVault OTX](https://otx.alienvault.com/), which comes prepackaged when downloading OpenSearch.
+- **Scan configuration**: Shows an overview of your scan configuration, including the **Log sources**, **Scan schedule**, and **Alert triggers** configured. From the **Actions** dropdown, you can also **Stop scan**, **Edit scan configuration**, or **Delete scan configuration**.
+
+
+## Step 1: Set up threat intelligence sources
+
+To add a threat intelligence source, select **Add threat intel source** from the threat intelligence page. The **Add custom threat intelligence source** page appears. 
+
+On the threat intelligence source page, add the following information:
+
+- **Name**: A name for the source.
+- **Description**: An optional description for the source.
+- **Threat intel source type**: The source type determines where the `STIX2` file is stored. You can choose one of the following options:
+  - **Remote data store location**: Connects to a custom data store. As of OpenSearch 2.16, only the `S3_SOURCE` type is supported. This setting also gives you the ability to set a download schedule, where OpenSearch downloads the newest STIX2 file from the data store. For more information, see [S3_SOURCE connection details](#s3_source-connection-details)
+  - **Local file upload**: Uploads a custom threat intelligence IOC file. Custom files cannot be set to download schedule and must be uploaded manually in order to update the IOCs. For more information, see  [Local file upload](#local-file-upload).
+- **Types of malicious indicators**: Determines the malicious IOCs to pull from the STIX2 file. The following IOCs are supported:
+  - IPV4-Address
+  - IPV6-Address
+  - Domains
+  - File hash
+
+After all the relevant information has been entered, select **Add threat intel source**.
+
+### Local file upload
+
+Local files uploaded as the threat intelligence source must use the following specifications:
+
+- Upload as a JSON file in the STIX2 format. For an example STIX2 file, [download the following example file]({{site.url}}{{site.baseurl}}/assets/examples/all-ioc-type-examples.json), which contains example formatting for all IOC types.
```

Comment:
@Naarcha-AWS 
> for all IOC types
Just to clarify, that file just contains examples for all of the types supported by this feature. There are other types of IOCs that aren't supported yet.

Reply:
```suggestion
- Upload as a JSON file in the STIX2 format. For an example STIX2 file, [download the following example file]({{site.url}}{{site.baseurl}}/assets/examples/all-ioc-type-examples.json), which contains example formatting for all supported IOC types.
```

Reply:
Added the word "supported" here.